app/actions/package_delete.rb

```diff
@@ -23,5 +19,17 @@ def delete(packages)
 
       []
     end
+
+    private
+
+    def create_package_source_deletion_job(package)
+      return Jobs::Runtime::BlobstoreDelete.new(package.guid, :package_blobstore) unless package_registry_configured?
+
+      package.bits? ? Jobs::Kubernetes::RegistryDelete.new(package.bits_image_reference) : nil
```

Comment:
What's the difference between Jobs::Kubernetes and Jobs::Runtime?

Reply:
We had put this class in a `jobs/kubernetes` folder and by convention the Ruby module namespacing should match the folder structure (we break this all over the place in Cloud Controller).

@matt-royal also has some hopes and dreams for a `kubernetes` module namespace and this is one step closer to that.